packages/react-components/src/organisms/__tests__/PeopleCard.test.tsx

```diff
@@ -30,6 +30,15 @@ it('renders the date in the correct format', () => {
     `"Joined: 1st January 2021"`,
   );
 });
+it('only show lab information if the user is on a lab', async () => {
+  const { queryByText, rerender } = render(
+    <PeopleCard {...props} displayName="John Doe" />,
```

Comment:
Don't rely on fixture data. Please set labs in the props. You don't need to set display name as that should be dealt with with the defaults in {...props} and we don't assert on it.